js/ide.js

```diff
@@ -7,7 +7,7 @@ import CodeMirror from 'codemirror/lib/codemirror.js';
 import moment from 'moment';
 import tokml from 'tokml';
 import togpx from 'togpx';
-import * as _saveAs from 'file-saver';
+import {saveAs} from 'file-saver';
```

Comment:
thanks. I couldn't find out how to get this to run properly on the weekend. I still have to get used to the new import syntax :man_shrugging: